functions/Get-DbaQueryStoreConfig.ps1

```diff
@@ -0,0 +1,176 @@
+		[string[]]$SqlServer,
+		[PsCredential]$Credential
+	)
+
+	DynamicParam {
+		if ($SqlServer) {
+			return Get-ParamSqlDatabases -SqlServer $SqlServer[0] -SqlCredential $Credential
+		}
+	}
+
+	BEGIN
+	{
+		# Convert from RuntimeDefinedParameter object to regular array
+		$databases = $psboundparameters.Databases
+		$exclude = $psboundparameters.Exclude
+
+        # Create a datatable to hold the Query Store configuration
+        $QSCDT = New-Object System.Data.DataTable
+
+        # Define the columns
+        $QSCDT_Instance = New-Object System.Data.DataColumn �Instance�,([string])
+        $QSCDT_Database = New-Object System.Data.DataColumn �Database�,([string])
+        $QSCDT_ActualState = New-Object System.Data.DataColumn �ActualState�,([string])
+        $QSCDT_FlushInterval = New-Object System.Data.DataColumn �FlushInterval�,([string])
+        $QSCDT_CollectionInterval = New-Object System.Data.DataColumn �CollectionInterval�,([string])
+        $QSCDT_MaxSize = New-Object System.Data.DataColumn �MaxSize�,([string])
+        $QSCDT_CurrentSize = New-Object System.Data.DataColumn �CurrentSize�,([string])
+        $QSCDT_CaptureMode = New-Object System.Data.DataColumn �CaptureMode�,([string])
+        $QSCDT_CleanupMode = New-Object System.Data.DataColumn �CleanupMode�,([string])
+        $QSCDT_StaleQueryThreshold = New-Object System.Data.DataColumn �StaleQueryThreshold�,([string])
+
+        # Add columns to the datatable
+        $QSCDT.Columns.Add($QSCDT_Instance)
+        $QSCDT.Columns.Add($QSCDT_Database)
+        $QSCDT.Columns.Add($QSCDT_ActualState)
+        $QSCDT.Columns.Add($QSCDT_FlushInterval)
+        $QSCDT.Columns.Add($QSCDT_CollectionInterval)
+        $QSCDT.Columns.Add($QSCDT_MaxSize)
+        $QSCDT.Columns.Add($QSCDT_CurrentSize)
+        $QSCDT.Columns.Add($QSCDT_CaptureMode)
+        $QSCDT.Columns.Add($QSCDT_CleanupMode)
+        $QSCDT.Columns.Add($QSCDT_StaleQueryThreshold)
+	}
+
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+            Write-Verbose "Connecting to $servername"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $servername -SqlCredential $Credential
+
+			}
+			catch
+			{
+				if ($SqlServer.count -eq 1)
+				{
+					throw $_
+				}
+				else
+				{
+					Write-Warning "Can't connect to $servername. Moving on."
+					Continue
+				}
+			}
+
+            $sqlVersion = $server.Version
+
+            $sqlVersion = $sqlVersion.ToString().Split(".")[0]
+
+            if($sqlVersion -ilt "13")
+                {
+
+                Write-Warning "The SQL Server Instance has a lower SQL Server version than SQL Server 2016. Skipping server."
+                break
+                }
+
+            
+            # We have to exclude all the system databases since they cannot have the Query Store feature enabled
+			$dbs = $server.Databases | Where-Object { $_.name -ne 'TempDb' -and $_.name -ne 'master' -and $_.name -ne 'msdb' -and $_.name -ne 'model' }
+
```

Comment:
Seem to recall we have an internal function for getting databases for a given instance...or maybe we are adding that for v1.0 of the module.